lib/plugins/aws/deploy/compile/events/schedule/index.js

```diff
@@ -38,9 +40,19 @@ class AwsCompileScheduledEvents {
               }
               ScheduleExpression = event.schedule.rate;
               State = event.schedule.enabled ? 'ENABLED' : 'DISABLED';
+              Input = event.schedule.input || '';
+              InputPath = event.schedule.inputPath || '';
```

Comment:
I am also realising that `Input` and `InputPath` are mutually exclusive, should we throw an error if both are set?